components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -94,6 +102,36 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
         }
       });
     } catch (error) {
+      console.error(error);
+      if (error instanceof ResponseError) {
+        switch (error.code) {
+          case ErrorCodes.USER_BLOCKED:
+            this.redirectTo(gitpodHostUrl.with({ pathname: "/blocked" }).toString());
+            return;
+          case ErrorCodes.SETUP_REQUIRED:
+            this.redirectTo(gitpodHostUrl.with({ pathname: "/first-steps" }).toString());
+            return;
```

Comment:
`/first-steps` looks like something ancient (ported over from [the old start page](https://github.com/gitpod-io/gitpod/blob/5d3f003a20beb516cf4f7434b835a5c8f069eb46/components/dashboard/src/components/start-workspace.tsx#L188-L190)). I guess this can be safely removed now, right?